Dockerfile

```diff
@@ -26,6 +27,7 @@ RUN cp /etc/ironic/ironic.conf /etc/ironic/ironic.conf_orig && \
     crudini --set /etc/ironic/ironic.conf deploy http_root /var/www/html/ && \
     crudini --set /etc/ironic/ironic.conf deploy default_boot_option local && \
     crudini --set /etc/ironic/ironic.conf inspector endpoint_override http://172.22.0.1:5050 && \
+    crudini --set /etc/ironic/ironic.conf json_rpc host_ip 172.22.0.1 && \
```

Comment:
Looks like we may need to update this, or change the host we connect to
`   ConnectFailure: Unable to establish connection to http://host11.beaker.tripleo.lab.eng.rdu2.redhat.com:8089: HTTPConnectionPool(host='host11.beaker.tripleo.lab.eng.rdu2.redhat.com', port=8089): Max retries
`
I changed the host_ip to 0.0.0.0 and the patch worked for me, but maybe it would be better to change so that we connect to the 172 address

Reply:
It will connect to whatever host name the conductor has. Can you check what hostname is used? If it's something `localhost`, then we can probably use `127.0.0.1` (could you try?).

Reply:
Oh, well, I'm apparently blind. So, `host11.beaker.tripleo.lab.eng.rdu2.redhat.com`. Okay, let's keep it 0.0.0.0 for now, I'll update.